tools/ci/runcodechecker.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+# tools/ci/runcodechecker.sh
```

Comment:
I don't see any place invoke this shell script

Reply:
https://github.com/apache/incubator-nuttx/blob/198e093ec2733dea7e818d84f346369cf7902f9e/.github/workflows/build.yml#L166-L175

It is called in `.github/workflows/build.yml`